docs/nima/pico/pico_intro.adoc

```diff
@@ -101,16 +175,26 @@ The Pico API provides types that are generally useful at compile time to assign
 
 == Configuration
 
-Pico Config builder
+Configuration can be achieved through a Pico add-on that is based on Helidon's configuration subsystem, and adds support for something called _config-driven services_ using the `@ConfiguredBy` annotation. 
 
+When applied to a target service interface, the `@ConfiguredBy` annotation will allow you to use a higher level aggregation for application configuration, and then allow the configuration to drive activation of services in the Pico Framework.
 
+NOTE: You can use Pico configuration in combination with the Helidon configuration subsystem. 
 
-== Examples
+There are a few additional caveats to understand about `ConfiguredBy` and its supporting infrastructure.
 
-//optional 
+* `@ConfigBean Builder` is used to aggregate configuration attributes to this higher-level, application-centric configuration beans.
 
-== Additional Information
+* The Pico Framework needs to be started with the supporting configdriven modules in order for configuration to drive service activation.
 
 
 
 
+== Examples
```

Comment:
When config-driven is used, then a different annotation processor should be used.

The basic pico service APT processor should use this one:

```
<build>
   <plugins>
       <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.10.1</version>
           <configuration>
               <annotationProcessorPaths>
                   <path>
                       <groupId>io.helidon.pico</groupId>
                       <artifactId>helidon-pico-processor</artifactId>
                       <version>${helidon.version}</version>
                   </path>
               </annotationProcessorPaths>
           </configuration>
       </plugin>
   </plugins>
</build>
```

whereas the Config-Driven one should use this (this one also covers the basic scenarios as well so you don't need both):

```
<build>
   <plugins>
       <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.10.1</version>
           <configuration>
               <annotationProcessorPaths>
                   <path>
                       <groupId>io.helidon.pico.configdriven</groupId>
                       <artifactId>helidon-pico-configdriven-processor</artifactId>
                       <version>${helidon.version}</version>
                   </path>
               </annotationProcessorPaths>
           </configuration>
       </plugin>
   </plugins>
</build>
```